README.md

```diff
@@ -0,0 +1,29 @@
+# Metatensor
```

Comment:
Aren't importing this somewhere in our docs? If yes, this will be broken now. If no we are fine.

Reply:
Fairly sure we don't import the README, only Contributing

Reply:
okay then I am fine.